apps/prairielearn/src/lib/file-store.ts

```diff
@@ -120,28 +132,28 @@ export async function deleteFile(file_id, authn_user_id) {
 /**
  * Option of returning a stream instead of a file
  *
- * @param {number | string} file_id - The file to get.
- * @return {Promise<(import('stream'))>} - Requested file stream.
+ * @param file_id - The file to get.
+ * @return Requested file stream.
  */
-export async function getStream(file_id) {
+export async function getStream(file_id: number | string): Promise<stream> {
   debug(`getStream(): file_id=${file_id}`);
   const file = await getFile(file_id, 'stream');
-  return file.contents;
+  return file.contents as stream /* Strange conversion */;
 }
 
 /**
  * Get a file from the file store.
  *
- * @param {number | string} file_id - The file to get.
- * @return {Promise<object>} An object with a buffer (of the file contents) and a file object.
+ * @param file_id - The file to get.
+ * @return An object with a buffer (of the file contents) and a file object.
  */
-export async function getFile(file_id, data_type = 'buffer') {
+export async function getFile(file_id: number | string, data_type = 'buffer') {
   debug(`get(): file_id=${file_id}`);
   const params = { file_id };
   const result = await sqldb.queryZeroOrOneRowAsync(sql.select_file, params);
   debug('get(): got row from DB');
 
-  let buffer, readStream;
+  let buffer: Buffer | GetObjectOutput['Body'], readStream: Buffer | GetObjectOutput['Body'];
```

Comment:
Nit, this is easier to read if you split the declaration into two lines.

```suggestion
  let buffer: Buffer | GetObjectOutput['Body'];
  let readStream: Buffer | GetObjectOutput['Body'];
```